client/src/components/search/search.jsx

```diff
@@ -21,14 +24,27 @@ class Search extends React.Component {
     }
   }
 
+  clearButton() {
+    this.setState({ value: '' });
+  }
+
   render() {
     const keywords = this.state.value;
     return (
       <div className={style.search}>
         <div className={style.glassContainer}>
           <div className={style.glass} />
         </div>
-        <input className={style.search_bar} value={keywords} onChange={this.handleChange} onKeyDown={this.pressEnter} placeholder="Search reviews" />
+        <input
+          className={style.search_bar}
+          value={this.state.value}
```

Comment:
Consider refactoring to use destructured state assignment, e.g. const const { value } = this.state.